explorer/e2e/anvil/anvil-filters.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { expect, test } from "@playwright/test";
+import {
+  filterRegex,
+  getFirstElementTextLocator,
+  testClearAll,
+  testFilterBubbles,
+  testFilterCounts,
+  testFilterPersistence,
+  testFilterPresence,
+} from "../testFunctions";
+import { anvilFilters, anvilTabs, anvilTabTestOrder } from "./anvil-tabs";
+
+const filter_index_list = [3, 4, 5, 10, 6, 2];
+const filter_index_list_short = [1, 10, 3];
```

Comment:
Would you be able to update these two to be MACRO_CASE to help indicate they are constants? Also, can you create constants for the integer values so it's more obvious what they are? For example:

```
const FILTER_ASSAY_INDEX = 3;
const FILTER_GENDER_INDEX = 4;

const FILTER_INDEX_LIST = [FILTER_ASSAY_INDEX, FILTER_GENDER_INDEX];
```

Reply:
Will do! Do you think it makes sense to store those constants in this file or in `anvilTabs.ts`?